packages/evm/contracts/court/controller/Controlled.sol

```diff
@@ -38,10 +50,28 @@ contract Controlled is IsContract, ConfigConsumer {
     }
 
     /**
-    * @dev Ensure the msg.sender is the DisputeManager module
+    * @dev Ensure the msg.sender is an active Voting module
+    */
+    modifier onlyActiveVoting() {
+        require(controller.isActive(VOTING, msg.sender), ERROR_SENDER_NOT_ACTIVE_VOTING);
+        _;
+    }
+
+    /**
+    * @dev Ensure the msg.sender is an active DisputeManager module
     */
-    modifier onlyDisputeManager() {
-        require(msg.sender == address(_disputeManager()), ERROR_SENDER_NOT_DISPUTES_MODULE);
+    modifier onlyActiveDisputeManagers() {
+        require(controller.isActive(DISPUTE_MANAGER, msg.sender), ERROR_SENDER_NOT_ACTIVE_DISPUTE_MANAGER);
+        _;
+    }
+
+    /**
+    * @dev Ensure the msg.sender is the current DisputeManager module
+    */
+    modifier onlyCurrentDisputeManager() {
+        (address addr, bool disabled) = controller.getDisputeManager();
+        require(msg.sender == addr, ERROR_SENDER_NOT_CURRENT_DISPUTE_MANAGER);
+        require(!disabled, ERROR_SENDER_NOT_ACTIVE_DISPUTE_MANAGER);
```

Comment:
> More of a side note, but going on the hypothetical here, is disabling the current dispute manager a valid state?

Yes, as a rule of thumb, I decided not to prevent anything from disabling/enabling modules, up to governor what to do, but everything is possible.

> is the main consequence to disallow new disputes, or would it cripple other parts of the system?

When you disable a module you're basically not allowing other modules to interact with it. If you disabled a dispute manager, even if it's the current one, you're blocking the dispute, guardians from voting, and their corresponding locked balances in the registry.